src/main/java/com/jalasoft/schoology/pages/Courses/CoursesAction.java

```diff
@@ -0,0 +1,16 @@
+package com.jalasoft.schoology.pages.Courses;
+
+import com.jalasoft.schoology.pages.AbstractPage;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class CoursesAction extends AbstractPage {
+
+    @FindBy(xpath = "//a[text()=\"Create Course\"]")
```

Comment:
use css instead xpath

Reply:
thanks Marco, but I have tried using css and the locator doesn't work